packages/std/src/mock.rs

```diff
@@ -256,14 +293,14 @@ pub fn mock_ibc_packet_recv<T: Serialize>(my_channel_id: &str, data: &T) -> StdR
             height: 12345678,
         }
         .into(),
-    })
+    }))
 }
 
 #[cfg(feature = "stargate")]
 /// Creates a IbcPacket for testing ibc_packet_{ack,timeout}. You set a few key parameters that are
 /// often parsed. If you want to set more, use this as a default and mutate other fields.
-/// The difference between mock_ibc_packet_recv is if `my_channel_id` is src or dest.
-pub fn mock_ibc_packet_ack<T: Serialize>(my_channel_id: &str, data: &T) -> StdResult<IbcPacket> {
+/// The difference from mock_ibc_packet_recv is if `my_channel_id` is src or dest.
+pub fn mock_ibc_packet<T: Serialize>(my_channel_id: &str, data: &T) -> StdResult<IbcPacket> {
```

Comment:
Is this still used anywhere in this repo? Looking at the comment

>  The difference from mock_ibc_packet_recv is if `my_channel_id` is src or dest.

it cannot easily be used for `mock_ibc_packet_ack` and `mock_ibc_packet_recv` at the same time. We can remove it or make it private.

Reply:
Yeah, fair. I'll make it private. On that note, should we mark `mock_ibc_channel` as deprecated for this release and make it private in the next one? Or just leave it?

Reply:
Let's make both private if not used directly. We can open up later on.

Reply:
The difference is `mock_ibc_channel` was there already, so people have potentially been using it.